MonoGame.Framework/Content/ContentReaders/ModelReader.cs

```diff
@@ -149,9 +149,6 @@ protected internal override Model Read(ContentReader reader, Model existingInsta
                 var parentBoneIndex = ReadBoneReference(reader, boneCount);
 				var boundingSphere = reader.ReadBoundingSphere();
 
-                // Tag
-                reader.ReadObject<object>();
-

```

Comment:
This looks broken to me.  You can't move the `ReadObject` call...  that has to remain where it is to obey the file format for models.

Did you test it with your own game?